webapp/src/components/systemConsole/surveyDateTime/surveyDateTime.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useState} from 'react';
+import {format, parse} from 'date-fns';
+
+import SurveyTimeSelector from 'components/surveyTimeSelector/surveyTimeSelector';
+import SurveyDateSelector from 'components/surveyDateSelector/surveyDateSelector';
+import {CustomComponentProps} from 'types/mattermost-webapp';
+
+import './style.scss';
+
+const defaultSurveyTime = '09:00';
+
+// date 30 days from today
+const defaultSurveyDate = new Date(new Date().setDate((new Date()).getDate() + 30));
```

Comment:
This is 30 days from the moment this file is loaded which seems like it might not be quite what you want. It may be better to save it in a ref in the component just in case someone leaves MM open for a few days.

Also, the JS `Date` type is awful to use, but I think you might be able to simplify this a bit to get rid of so many date constructors and brackets if you break it into two lines
```js
const defaultSurveyDate = new Date();
defaultSurveyDate.setDate(defaultSurveyDate.getDate() + 30);
```

Reply:
Fixed. Set the state to 30 days from today if a saved config is not found.